src/hooks/useTabActive/useTabActive.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react'
+
+function useTabActive() {
```

Comment:
This is not necessary at all, because react query is configured and [doesn't make queries](https://tanstack.com/query/v4/docs/guides/window-focus-refetching
) when the tab is not active :)